src/algokit_utils/_transfer.py

```diff
@@ -67,18 +82,48 @@ def transfer(client: "AlgodClient", parameters: TransferParameters) -> PaymentTx
         note=parameters.note.encode("utf-8") if isinstance(parameters.note, str) else parameters.note,
         sp=suggested_params,
     )  # type: ignore[no-untyped-call]
+
+    result = send_transaction(client=client, transaction=transaction, parameters=parameters)
+    assert isinstance(result, PaymentTxn)
+    return result
+
+
+def transfer_asset(client: "AlgodClient", parameters: TransferAssetParameters) -> AssetTransferTxn:
+    xfer_txn = AssetTransferTxn(
+        sp=client.suggested_params(),
+        sender=parameters.from_account,
```

Comment:
Should we be using `address_from_private_key(parameters.from_account.private_key)` like what `transfer` does?